stubs/inertia/resources/js/Pages/Auth/VerifyEmail.vue

```diff
@@ -18,7 +18,7 @@
                     Resend Verification Email
                 </jet-button>
 
-                <inertia-link :href="route('logout')" method="post" class="underline text-sm text-gray-600 hover:text-gray-900">Logout</inertia-link>
+                <inertia-link :href="route('logout')" method="post" as="button" class="underline text-sm text-gray-600 hover:text-gray-900">Logout</inertia-link>
```

Comment:
I haven't had a chance to install and test this yet, but can you confirm that this hasn't messed up the style of the Logout button? In Ping CRM I had to add `w-full text-left` to get it render properly, just because of default button browser styles.

Reply:
Yep, looks fine 👍